contrib/experimental_dict_builders/benchmarkDictBuilder/benchmark.c

```diff
@@ -379,17 +414,21 @@ int main(int argCount, const char* argv[])
         result = 1;
         goto _cleanup;
       }
+
+      k = fastParam.k;
+      d = fastParam.d;
     }
 
-    /* for fastCover (with k provided) */
+
+    /* for fastCover (with k and d provided) */
     {
       ZDICT_fastCover_params_t fastParam;
       memset(&fastParam, 0, sizeof(fastParam));
       fastParam.zParams = zParams;
       fastParam.splitPoint = 1.0;
-      fastParam.d = 8;
+      fastParam.d = d;
       fastParam.f = f;
-      fastParam.k = 200;
+      fastParam.k = k;
```

Comment:
Does FASTCOVER select approximately the same `k` value as COVER? Does `d=6` or `d=8` win more often, or is it about evenly split?

Reply:
For hg-changelog, COVER chooses k =98, which is the k value for most f values of FASTCOVER. For github, FASTCOVER with all f values chooses k=1778, which is very different from l=1298 chosen by COVER. For hg-commands and hg-manifest,  the k value varies greatly across different f values, so k value for some f values is close to COVER while others vary greatly. COVER chooses d=8 for all 4 sample sets. d=8 always wins for github and hg-changelog. For hg-changelog, d=8 wins more than d=6. For hg-manifest, there's an even split between d=8 and d=6.